CONTRIBUTING.md

```diff
@@ -30,23 +30,10 @@ Do you want to **contribute a failing test**? [This tutorial will show you how](
 
 ## Preparing Pull Request
 
-3 steps will make your pull-request easy to merge:
-
 - **1 feature per pull-request**
 - **new features need tests**
-- CI must pass... you can mimic it locally by running
-
-    ```bash
-    composer complete-check
```

Comment:
The script is exists, contributing is to rector-src which the composer.json consist the script, see 

https://github.com/rectorphp/rector-src/blob/73aa7e3d68d74fc0b9b26fc543bf1207cfc8d0dd/composer.json#L116

Reply:
Your right! is there a reason why it isn't in https://github.com/rectorphp/rector/blob/main/composer.json ?

I was working there on a fix then found out i should fix it elsewhere. But these commands are not in rectorphp/rector

Reply:
rectorphp/rector is generated content, you need to contribute it here instead.

Reply:
Okay then we can close this pull request and this issue: https://github.com/rectorphp/rector/issues/7477